scripts/build/build/targets.py

```diff
@@ -204,6 +204,30 @@ def BuildHostTarget():
     target.AppendModifier('googletest', use_googletest=True).OnlyIfRe('-tests')
     target.AppendModifier('terms-and-conditions', terms_and_conditions_required=True)
     target.AppendModifier('webrtc', enable_webrtc=True)
+    target.AppendModifier('unified', unified=True).OnlyIfRe(
+        "(-" + '-|-'.join([
```

Comment:
Maybe add a comment for an example that this creates something like:

`(-air-purifier-|-all-clusters-|-bridge-|-....-|-water-leak-detector-)`

Realistically this seems the same as `-(air-purifier|all-clusters|....|water-leak-detector)-` so I wonder if we could instead do:

`"-(" + "|".join([.....]) + ")-"` ... given that that seems both shorter regex and shorter strings in the join.